app/src/main/kotlin/com/simplemobiletools/clock/receivers/AlarmReceiver.kt

```diff
@@ -31,8 +34,9 @@ class AlarmReceiver : BroadcastReceiver() {
             if (isOreoPlus()) {
 
                 val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
-                if (notificationManager.getNotificationChannel("Alarm") == null) {
-                    NotificationChannel("Alarm", "Alarm", NotificationManager.IMPORTANCE_HIGH).apply {
+                if (notificationManager.getNotificationChannel(ALARM_NOTIFICATION_CHANNEL_ID) == null) {
+                    oldNotificationChannelCleanup(notificationManager) // cleans up previous notification channel that had sound properties
```

Comment:
have you tested this with multiple alarms with different audio being set up too?

Reply:
Yes I have